src/routes/api/v1/index.ts

```diff
@@ -0,0 +1,22 @@
+import { Router } from 'express'
+import SubscriptionsController from '../../../controllers/api/v1/subscriptions'
+import {
+  subscribeWalletTransfersToValidation,
+  unsubscribeWalletTransfersToValidation
+} from '../../../controllers/api/v1/subscriptions/validations'
+
+const router = Router()
+
+router.post(
+  '/subscribe/wallet/transfers-to',
```

Comment:
IMO `/wallet` part of the route is not needed. We can keep it as:
`subscribe/:eventName`, like `subscribe/transfers-to`